plugins/modules/foreman_operatingsystem.py

```diff
@@ -190,7 +206,24 @@ def main():
             name=dict(),
             release_name=dict(),
             description=dict(),
-            family=dict(),
+            family=dict(choices=[
```

Comment:
we have the same list in `plugins/modules/foreman_installation_medium.py` already, so I wonder if we should define it in `foreman_helper` once and reference from here?

Reply:
From my perspective, absolutely! Will do so, thank you for the suggestion :)